### PR TITLE
Start adding urgent care clinics in the US

### PR DESCRIPTION
**Pull request by tas50**
Due to our wonderfully backwards health care system there's a big
business for private care urgent care clinics. I added ZoomCare which is
a big one in Portland & Seattle. This could probably be expanded to
include some of the clinics here: https://www.healthdatamanagement.com/list/30-largest-urgent-care-center-chains-in-healthcare

Signed-off-by: Tim Smith <tsmith@chef.io>

### CODE REVIEW

File: brands/healthcare/clinic.json [hunk 1]
```diff
@@ -0,0 +1,13 @@
+{
+  "healthcare/clinic|ZoomCare": {
```
Comment (1ec5) [inline brands/healthcare/clinic.json]: (Note that a `matchNames` of `zoom+care` is unnecessary because plus signs are removed in fuzzy matching.)

Comment (tas50) [reply brands/healthcare/clinic.json]: Yeah I figured that would just magically happen here

File: brands/healthcare/clinic.json [hunk 2]
```diff
@@ -0,0 +1,13 @@
...
+    "countryCodes": ["us"],
+    "tags": {
+      "amenity": "clinic",
+      "brand": "ZoomCare",
+      "brand:wikidata": "Q64120374",
+      "brand:wikipedia": "en:ZoomCare",
+      "healthcare": "clinic",
```
Comment (1ec5) [inline brands/healthcare/clinic.json]: `urgent_care=yes` `healthcare:speciality=emergency` seems like an appropriate way to differentiate this kind of clinic from retail clinic and community health centers without resorting to `emergency=yes` (which is for EMS admission).

Comment (maxerickson) [reply brands/healthcare/clinic.json]: I think it's better to reserve healthcare:speciality=emergency for actual emergency rooms, some urgent care facilities aren't doing much more than primary care.

Glancing at their website, it appears some of their locations do provide a high level of service, but not all of them, and they still say to go to the ER if it is a real emergency.

Comment (tas50) [reply brands/healthcare/clinic.json]: So Zoom Care isn't just an urgent care sort of clinic. You can just use them as a normal doctor if you want. urgent_care=yes might make sense, but it's def not emergency specialty. It's people with colds and also just people getting physicals and yearly checkups.

Comment (tas50) [reply brands/healthcare/clinic.json]: Source: ZoomCare is my go to when I'm sick and don't want to head across town to my regular doctor.

Comment (maxerickson) [reply brands/healthcare/clinic.json]: urgent_care=yes is meant to indicate that walk-in type service is available, not that it is the only type of service available. I think I'm the main user of the tag (but I'm bad at documenting things on the wiki or asking tagging if they want to argue about things). https://overpass-turbo.eu/s/Jt5

Comment (tas50) [reply brands/healthcare/clinic.json]: I meant above that `urgent_care=yes` would be appropriate here since that is a big part of what they offer.

**Pull request merged**